File: Feitos em Java/2702.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        int[] pedidos = new int [3];
        int[] refeicoes = new int [3];
        int faltou=0;
        refeicoes[0] = leitor.nextInt();
        refeicoes[1] = leitor.nextInt();
        refeicoes[2] = leitor.nextInt();
        pedidos[0] = leitor.nextInt();
        pedidos[1] = leitor.nextInt();
        pedidos[2] = leitor.nextInt();
	for(int i=0;i<3;i++){
            if(pedidos[i]>refeicoes[i]){
                faltou+=(pedidos[i]-refeicoes[i]);
            }
	}
        System.out.println(faltou);
    }
}

File: Feitos em C/2702.c
#include <stdio.h>
int main() {
	int pedidos[3],refeicoes[3],faltou=0,i;
	scanf("%d",&refeicoes[0]);
	scanf("%d",&refeicoes[1]);
	scanf("%d",&refeicoes[2]);
	scanf("%d",&pedidos[0]);
	scanf("%d",&pedidos[1]);
	scanf("%d",&pedidos[2]);
	for(i=0;i<3;i++){
		if(pedidos[i]>refeicoes[i]){
			faltou+=(pedidos[i]-refeicoes[i]);
		}
	}
	printf("%d\n",faltou);
	return 0;
}
